### ISSUE TITLE
Random failures in the schedule_test.rb integration test

### ISSUE THREAD

**Issue by stuzart** [seek4science/seek#1917]
Happens quite frequently:

```
ScheduleTest#test_news_feed_refresh_changes_with_config [/home/runner/work/seek/seek/test/integration/schedule_test.rb:101]
Minitest::Assertion: Expected: [731 minutes]
  Actual: [30 minutes]
F
Failure:
ScheduleTest#test_should_offset_daily_job_runtime_by_configured_amount [/home/runner/work/seek/seek/test/integration/schedule_test.rb:119]
Minitest::Assertion: Expected: [4 hours, {:at=>"1:43am"}]
  Actual: [4 hours, {:at=>"1:00am"}]
```

**Comment by stuzart** [comment]
caused by config caching.

**Comment by stuzart** [comment]
Fix didn't work, failed again: https://github.com/seek4science/seek/actions/runs/9500183070/job/26182660279

**Comment by stuzart** [comment]
should now be fixed by #2004
